lib/crypto.js

```diff
@@ -83,6 +83,7 @@ exports.createHmac = exports.Hmac = Hmac;
 function Hmac(hmac, key, options) {
   if (!(this instanceof Hmac))
     return new Hmac(hmac, key, options);
+  this.key = key;
```

Comment:
Should be either hidden, documented, or at least prefixed. Not sure if prefixing is a viable solution nowdays.